static/src/javascripts/projects/common/modules/charts/doughnut.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+/*
+ * Inspired by
+ * - http://www.chartjs.org/
+ * - http://codepen.io/githiro/pen/ICfFE
+ * - https://github.com/mbostock/d3/blob/master/src/svg/arc.js
+ */
+import $ from 'lib/$';
+import bonzo from 'bonzo';
```

Comment:
If you only need `bonzo` for the types then can you import only the type to make this explicit?